schedule/yast/yast2_ncurses_gnome.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name:           yast2_ncurses_gnome
+description:    >
+    Test for yast2 UI, ncurses only. Running on created gnome images
+    which provides both text console for ncurses UI tests as well as
+    the gnome environment for the GUI tests.
+schedule:
+    - "{{uefi_boot}}"
+    - "{{bootloader_start}}"
+    - boot/boot_to_desktop
+    - console/prepare_test_data
+    - console/consoletest_setup
+    - "{{all_modules}}"
+    - "{{modules_from_default_installation}}"
+conditional_schedule:
+    uefi_boot:
+        ARCH:
+            aarch64:
+            - boot/uefi_bootmenu
+    bootloader_start:
+        ARCH:
+            s390x:
+            - installation/bootloader_start
+    all_modules:
+        ARCH:
+            x86_64:
+                - console/yast2_rmt
+                - console/yast2_ntpclient
+                - console/yast2_tftp
+                - console/yast2_proxy
+                - console/yast2_vnc
+                - console/yast2_samba
```

Comment:
i think this should be excluded due to https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10270. Could you double check?

Reply:
good that you notice, but let's wait for @JRivrain if it was able to regenerate that certificate.

Reply:
good catch, I will delete this line so @JRivrain can decide if re-enable or not depending on result.